resources/public/api/conf/2.0/testdata/annual-return.md

```diff
@@ -407,11 +407,11 @@
 }
 </pre>
             </td>
-            <td><p>HTTP status: <code class="code--slim">406 (Not Acceptable)</code></p>
```

Comment:
need to check this 406 is still a valid error

Reply:
@fraser-sprott according to the documented errors, it should be replaced:
https://confluence.tools.tax.service.gov.uk/pages/viewpage.action?spaceKey=ELSE&title=Table+of+Test+Data+Errors+Only

Reply:
Yeah, I agree that it's been mentioned to change, but it's still a reachable error code which makes me think somethings been missed as part of that investigation, [(header validation)](https://github.com/hmrc/lisa-api/blob/0531e197949a55b94d1663e133e71ffc49fb4f25/app/uk/gov/hmrc/lisaapi/controllers/APIVersioning.scala#L60), will catch up with you later to look into this one